### PR TITLE
Fix layout of `service-account.md`

### LINKED ISSUE #2415: `GOOGLE_CREDENTIALS` or `GOOGLE_APPILICATION_CREDENTIALS`?
File: [themes/default/content/docs/intro/cloud-providers/gcp/service-account.md](https://www.pulumi.com/docs/intro/cloud-providers/gcp/service-account/)

the [GCP docs](https://cloud.google.com/docs/authentication/production#manually) suggest setting the env var `GOOGLE_APPLICATION_CREDENTIALS` - are the pulumi docs out of date?

### PR DESCRIPTION
**Pull request by iwahbe**
## Description

> [!IMPORTANT]
> The first commit is just cleanup. The second commit fixes the cosmetics. The [3rd commit](https://github.com/pulumi/registry/pull/6201/commits/8f55cade4b6101f07c6a2812c64d35ea0b853134) fixes the associated issue.

This changes the appearance of `/registry/packages/gcp/service-account/`.

Before:

Screenshot 2024-12-10 at 7 18 52 PM

After:

Screenshot 2024-12-10 at 7 17 42 PM

Fixes #2415

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,9 +33,12 @@ build:
 	$(MAKE) build-assets
 	./scripts/build.sh
 
+HUGO_SERVE := hugo serve --buildDrafts --buildFuture --ignoreVendorPaths="github.com/pulumi/registry/**/*" | grep -v -e 'WARN .* REF_NOT_FOUND'
```
Comment (guineveresaenger) [inline Makefile:36]: We don't need to specify a port?

File: Makefile [hunk 2]
```diff
@@ -60,7 +63,8 @@ serve-assets:
 
 .PHONY: serve-all
 serve-all:
-	./node_modules/.bin/concurrently --kill-others -r "./scripts/serve.sh" "yarn --cwd ./themes/default/theme run start"
```
Comment (guineveresaenger) [inline Makefile:63]: What's the rationale behind removing
`./node_modules/.bin/concurrently --kill-others` ?

**Review by guineveresaenger — APPROVED** [review]
Couple questions about the `make serve` changes but LGTM.

File: Makefile (hunk 2 above)
Comment (iwahbe) [reply Makefile:63]: Just trying to drop the dependency. I can put it back: https://github.com/pulumi/registry/pull/6201/commits/29ca99d7df6a8c8348360ad57e682f1965c82d1e.

File: Makefile (hunk 1 above)
Comment (guineveresaenger) [reply Makefile:36]: dope, makes sense!

**Pull request merged**